index.js

```diff
@@ -195,10 +195,10 @@ class NodeClam {
                 // If scanner binary doesn't exist...
                 if (!await this._is_clamav_binary(this.scanner)) {
                     // Fall back to other option:
-                    if (this.scanner == 'clamdscan' && this.settings.clamscan.active === true && await this._is_clamav_binary('clamscan')) {
-                        this.scanner == 'clamscan';
```

Comment:
The change from comparison to assignment in this line and line 201 are the main fixes. The other changes are for consistency.